ouroboros-consensus-cardano-test/src/Test/ThreadNet/Infra/ShelleyBasedHardFork.hs

```diff
@@ -128,7 +129,7 @@ type ShelleyBasedHardForkConstraints era1 era2 =
   , SL.TranslationError   era2 SL.NewEpochState  ~ Void
   , SL.TranslationError   era2 SL.ShelleyGenesis ~ Void
 
-  , SL.TranslationContext era2 ~ ()
+  , SL.TranslationContext era1 ~ ()
```

Comment:
This is not true for `era1 ~ AlonzEra`, but that's OK. At first, we'll only ever have `era2 ~ AlonzoEra`.